### DISCUSSION TITLE
How do I change the default "save" keyboard shortcut?

### DISCUSSION THREAD

**Discussion by janek** [omnivore-app/omnivore#4014] [Q&A]
For me it's `OPTION + O` and I'm not sure if I set it myself or not. But I searched the interface, the docs, in google and a bit of github and strangely can't find the possibility to change it. Can you tell me how? When I type in my native language I use `OPTION + O` for a letter, so it gets in the way quite a lot.

I'm on macOS and Chromium (Arc)

**Comment by finn-matti** [comment]
And is there even one for the safari extension? It seems like it also should be according to [this file](https://github.com/omnivore-app/omnivore/blob/main/apple/Sources/SafariExtension/Resources/manifest.json#L90), but it doesn't seem to do anything.
